ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/PartyMapper.java

```diff
@@ -70,32 +71,48 @@ public CCDParty to(Party party) {
     }
 
     @Override
-    public Party from(CCDParty ccdParty) {
+    public Party from(Object party) {
+        CCDParty ccdParty = (CCDParty) party;
         switch (ccdParty.getType()) {
             case COMPANY:
                 CCDCompany ccdCompany = ccdParty.getCompany();
-                return new Company(ccdCompany.getName(), addressMapper.from(ccdCompany.getAddress()),
+                return new Company(ccdCompany.getName(),
+                    addressMapper.from(ccdCompany.getAddress()),
                     addressMapper.from(ccdCompany.getCorrespondenceAddress()),
-                    ccdCompany.getPhoneNumber(), representativeMapper.from(ccdCompany.getRepresentative()),
+                    ccdCompany.getPhoneNumber(),
+                    representativeMapper.from(ccdCompany.getRepresentative()),
                     ccdCompany.getContactPerson());
             case INDIVIDUAL:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 10 to at most 5, for example by extracting code into methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1151)